dashboard/app/views/helpful_links/index.html.haml

```diff
@@ -0,0 +1,14 @@
+%h1= 'Helpful Links'
+
+%h2 Reference Guides
+%ul
+  %li= link_to 'CSD', '/courses/csd-2022/guides/'
```

Comment:
Will these need to be updated every year?

Reply:
I was wondering about this... I can ask about how/when these pages are updated

Reply:
fwiw, this is definitely good for now so the work to use /course/csd/guides can definitely be a follow up PR/ticket

Reply:
https://codedotorg.atlassian.net/browse/TEACH-50 I created the follow up ticket!